Manual.md

```diff
@@ -781,6 +781,17 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gem` For packages that are installed using gemspec files or gems
+from rubygems. The build style automatically sets `distfiles` to the
+rubygems name of the package if not set manually. The `gem_cmd`
+variable can be used to set the name of the gem binary.
+
+The build style requires the `pkgname` variable to be their name on
+RubyGems with ruby- prefixed. The `distfiles` location is automatically
+set, ruby is added to `hostmakedepends` and `depends`, and the build
+process is extensively modified to not use a `wrksrc` but instead call
+`gem` into the downloaded gem file directly.
```

Comment:
Hm, how would one go about doing custom phases then? Would one still go into ${wrksrc} e.g. to do sed calls?